resident/resident-service/src/main/java/io/mosip/resident/util/EncryptorUtil.java

```diff
@@ -85,7 +85,9 @@ public String encrypt(byte[] data, String refId) {
                 ServiceError error = (ServiceError) responseDto.getErrors().get(0);
                 throw new PacketDecryptionFailureException(error.getMessage());
             }
-
+            if(responseDto != null || responseDto.getResponse() != null) {
+                DecryptResponseDto decryptResponseDto = mapper.convertValue(responseDto.getResponse(), DecryptResponseDto.class);
+            }
```

Comment:
Please update the code as per the discussion.

Reply:
Yes updated.